packages/discovery-provider/src/api/v1/models/tracks.py

```diff
@@ -126,6 +127,47 @@
     },
 )
 
+
+tip_gate = ns.model("tip_gate", {"tip_user_id": fields.String(required=True)})
+follow_gate = ns.model("follow_gate", {"follow_user_id": fields.String(required=True)})
+nft_collection = ns.model(
+    "nft_collection",
+    {
+        "chain": fields.String(enum=["eth", "sol"], required=True),
+        "address": fields.String(required=True),
+        "name": fields.String(required=True),
+        "imageUrl": fields.String(),
+        "externalLink": fields.String(),
+    },
+)
+nft_gate = ns.model(
+    "nft_gate", {"nft_collection": fields.Nested(nft_collection, required=True)}
+)
+usdc_gate = ns.model(
+    "usdc_gate",
+    {
+        "splits": fields.Wildcard(fields.Integer, required=True),
```

Comment:
Just want to confirm we actually store/return the splits as numbers and not strings that get parsed into BigInt-type structures.